packages/office-addin-lint/src/lint.ts

```diff
@@ -24,8 +25,10 @@ function normalizeFilePath(filePath: string): string {
 }
 
 function getEsLintBaseCommand(useTestConfig: boolean = false): string {
-  const configFilePath = useTestConfig ? eslintTestConfigPath : eslintConfigPath;
-  const eslintBaseCommand: string = `node "${eslintFilePath}" -c "${configFilePath}" --resolve-plugins-relative-to "${__dirname}"`;
+  const projLintConfig = path.resolve(process.cwd(), ".eslintrc.json");
+  const libLintConfig = useTestConfig ? eslintTestConfigPath : eslintConfigPath;
+  const useConfig = fs.existsSync(projLintConfig) ? "" : `-c ${libLintConfig} `;
```

Comment:
If `useTestConfig` is true, don't we want to use the `eslintTestConfigPath`? It is weird to pass a parameter `useTestConfig` to the function that may not be used at all depending on if a file exists.
Maybe this file checking can be done in another place? Or this boolean handling somewhere else?